api/v1_docs.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+include_once($relPath.'faq.inc');
+
+function api_v1_documents(string $method, array $data, array $query_params): array
+{
+    global $external_faq_overrides;
+
+    $lang_code = $query_params["lang_code"] ?? null;
+    if ($lang_code) {
+        $docs_with_lang = [];
+        foreach ($external_faq_overrides as $doc => $doc_array) {
+            if (isset($doc_array[$lang_code])) {
+                $docs_with_lang[] = $doc;
+            }
+        }
+        return $docs_with_lang;
+    } else {
+        return array_keys($external_faq_overrides);
+    }
+}
+
+function api_v1_document(string $method, array $data, array $query_params): string
```

Comment:
This function needs to accept a locale so it knows which URL to return. And `get_faq_url()` below _must_ be passed a locale -- either the one specified by the user or `en_US` so that it returns English if none was specified. We should not have generalized endpoints rely on the user's preferences like this one does (and @srjfoo discovered with testing).

Reply:
Anyhow, I've changed it as you suggested. It also seems rather unsatisfactory that _GET should be referenced inside the function get_desired_language() but with the language code specified we won't ever get there.